### ISSUE TITLE
Will be LHC fully comply with GDPR?

### ISSUE THREAD

**Issue by black23** [LiveHelperChat/livehelperchat#1161]
Hello, 
GDPR will start to apply in May 2018. More informations [here.](http://ec.europa.eu/justice/data-protection/reform/files/regulation_oj_en.pdf). LHC can collect personal data like IP adress, e-mail etc. If we want to use LHC, we need some tools and settings. For exapmplet anonymisation of IP, automated data deletion in time period or log for user cofirmation of agree with data processing. In this time LHC has some of this tools, but not all. Do you plan to make LHC fully comply with GDPR? It's very importat for all EU users.
Thank you.

**Comment by remdex** [comment]
Well,
Term and conditions possible to set now. To avoid tracking IP or just encrypt it possible to make an option in settings. Just will need a little time to know what exactly has to be done :)

**Comment by black23** [comment]
Results of my research, some of GDPR related changes:

- stop tracking of IP's if needed (is it possibe now?)
- make IP anonymised by settings for example 234.15.xxx.xxx just for country detection
- CRON that anonymised talk records after x days after close = it should be saved just for statistiscs purposes
- anonymise tool on demand from filtered list of talks (now we have just Delete button)

**Comment by black23** [comment]
@remdex  Any news in this case? THX

**Comment by remdex** [comment]
I'll be working on this task this weekend :)

**Comment by skiner68** [comment]
I do not think this anonymize solution is sufficient. We need to anonymize all user data (name, email, phone). I also have not found any option to delete complete chat records older than x days in documentation. I would appreciate any help on this.
